tmt/utils.py

```diff
@@ -2906,6 +2917,14 @@ class ListFormat(enum.Enum):
 _FORMAT_VALUE_LIST_ENTRY_INDENT = '  - '
 
 
+def assert_window_size(window_size: Optional[int]) -> None:
```

Comment:
A short (oneline) docstring explaining motivation for the assert?

Reply:
Added in c6285af6